<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>listfordoctor</class>
 <widget class="QDialog" name="listfordoctor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>345</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Gargi</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Записи к варчам</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>600</width>
     <height>171</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_7">
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="lbSpecialization_16">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Gargi</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Специальность</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="leFio">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="leSpecialization">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lbSpecialization_12">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Gargi</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ФИО врача</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lbSpecialization_17">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>10</y>
     <width>650</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>650</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>650</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Gargi</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Текущий врач</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRecordings">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>290</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Посмотреть записи</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>-60</x>
     <y>60</y>
     <width>741</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="btnBack_3">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btnPrevious_3">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Предыдущая</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btnNext_3">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Следующая</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
